js/metrics_test.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright 2018 the Deno authors. All rights reserved. MIT license.
+import { test, assert } from "./test_util.ts";
+import * as deno from "deno";
+
+test(function metrics() {
+  // TODO: metrics command does not send any data buffer to privileged side
+  // should we call other method that does to present it here?
```

Comment:
@ry what do you think about that? Should we just leave it as is now?

Reply:
Sorry, I don't understand the TODO. Can you rephrase?

Reply:
Bingo